project.clj

```diff
@@ -1,58 +1,56 @@
   :url "http://github.com/puppetlabs/trapperkeeper-metrics"
 
+  :min-lein-version "2.7.1"
+
   :pedantic? :abort
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.4"]
+                   :inherit [:managed-dependencies]}
+
+  :dependencies [[org.clojure/clojure]
 
                  ;; begin version conflict resolution dependencies
-                 [clj-time "0.11.0"]
-                 [commons-codec "1.9"]
-                 [org.clojure/tools.macro "0.1.5"]
-                 [org.clojure/tools.reader "1.0.0-alpha1"]
-                 [prismatic/schema "1.1.0"]
-                 [slingshot "0.12.2"]
-                 [commons-io "2.4"]
-                 [ring/ring-servlet "1.4.0"]
+                 [clj-time]
+                 [commons-codec]
+                 [org.clojure/tools.macro]
+                 [org.clojure/tools.reader]
+                 [prismatic/schema]
+                 [slingshot]
+                 [commons-io]
+                 [ring/ring-servlet]
                  ;; end version conflict resolution dependencies
 
-                 [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/trapperkeeper ~tk-version]
-                 [puppetlabs/ring-middleware "1.0.0"]
-
-                 [ring/ring-core "1.4.0"]
-
-                 [cheshire "5.6.1"]
-                 [org.clojure/java.jmx "0.3.1"]
-                 ;; ring-defaults brings in a bad, old version of the servlet-api, which
-                 ;; now has a new artifact name (javax.servlet/javax.servlet-api).  If we
-                 ;; don't exclude the old one here, they'll both be brought in, and consumers
-                 ;; will be subject to the whims of which one shows up on the classpath first.
-                 ;; thus, we need to use exclusions here, even though we'd normally resolve
-                 ;; this type of thing by just specifying a fixed dependency version.
-                 [ring/ring-defaults "0.1.5" :exclusions [javax.servlet/servlet-api]]
+                 [puppetlabs/kitchensink]
+                 [puppetlabs/trapperkeeper]
+                 [puppetlabs/ring-middleware]
+
+                 [ring/ring-core]
+
+                 [cheshire]
+                 [org.clojure/java.jmx]
+                 [ring/ring-defaults]
                  ;; Explicitly reference the correct servlet-api so that downstream
                  ;; projects will always get it
                  [javax.servlet/javax.servlet-api "3.1.0"]
 
-                 [org.clojure/tools.logging "0.3.1"]
-                 [org.slf4j/slf4j-api "1.7.13"]
+                 [org.clojure/tools.logging]
+                 [org.slf4j/slf4j-api]
                  [io.dropwizard.metrics/metrics-core "3.1.2"]
-                 [puppetlabs/comidi "0.3.1"]
-                 [puppetlabs/i18n "0.4.1"]]
+                 [puppetlabs/comidi]
+                 [puppetlabs/i18n]]
+
+  :plugins [[puppetlabs/i18n "0.4.1"]
```

Comment:
Ah, good catch. I'll revise the patch to bump that.

Related note: would it be possible to inherit this plugin from clj-parent? Otherwise, I imagine the two i18n versions will drift fairly often.